include/lsst/TMPL/ExampleOne.h

```diff
@@ -0,0 +1,173 @@
+// -*- lsst-c++ -*-
+
+/*
+ * LSST Data Management System
+ * Copyright 2008-2016  AURA/LSST.
```

Comment:
Please consider updating the copyright years in the C++ examples (or using the "see copyright file" if that's allowed).